plugins/module_utils/network/common/netconf.py

```diff
@@ -159,7 +159,7 @@ def remove_namespaces(data):
             "It can be installed using `pip install ncclient`"
         )
     return NCElement(
-        to_text(data.strip(), errors="surrogate_then_replace"),
```

Comment:
This fix reverts the original change and introduced the fixed issue again https://github.com/ansible-collections/ansible.netcommon/issues/151

Reply:
Can you please test the original issue with this PR?

Reply:
@ganeshrn yes I will check this

Reply:
> This fix reverts the original change and introduced the fixed issue again #151

This will not revert the changes as We are still using to_text function call.
only thing which is fixed here is strip() call that should be on the object returned from to_text method instead  of data

Reply:
Cool thanks for working on it.